tests/clar.c

```diff
@@ -440,21 +446,24 @@ void
 clar_test_shutdown(void)
 {
 	clar_print_shutdown(
+		_clar.evaluation_duration,
 		_clar.tests_ran,
-		(int)_clar_suite_count,
-		_clar.total_errors
+		_clar.suites_ran,
+		_clar.total_errors,
+		_clar.total_skipped
 	);
 
 	clar_unsandbox();
 }
 
 int
-clar_test(int argc, char **argv)
+clar_test(void)
```

Comment:
I'm still on the edge regarding this change. Just assuming that clar might be used elsewhere (which I don't really know of) this would break the API for them. So I'm opposed to changing the signature here without us (that is, the libgit2 project) having decided whether to diverge from upstream or not. The git__timer code is obviously already diverging, but that's just a minor thing compared to also breaking the API.